docs/user/app_getting_started.md

```diff
@@ -146,48 +146,102 @@ After clicking **Create**, you will be presented with a newly rendered floor pla
 
 ### Tile management on the Floor Plan
 
-To edit information on a tile, either click the:
-* Yellow pencil button in the upper left to edit it.
-* Box within the tile to view the existing rack assigned.
-* Green **"+"** icon in the center of any rectangle in the grid.
-* The blue **"+ Add Tile"** button in the upper right of the floor plan.
-
-![Add Tile form part 1](../images/add-tile-form.png)
+To add a tile, either click the:
 
+- Green **"+"** icon in the center of any rectangle in the grid.
+- The blue **"+ Add Tile"** button in the upper right of the floor plan.
 
-### Floorplan Navigation
+To edit information on a tile, either click the:
 
-If you've defined a large floor plan or have a small display, you can use your mouse wheel to zoom in for a better view. While zoomed in, click and drag to pan around the grid.
+- Yellow pencil button in the upper left to edit it.
+- Box within the tile to view the existing object assigned.
 
 ### Tile Options
 
 For each tile, you can:
 
 - **Assign a Status**: Choose from the predefined Statuses (e.g., "Active", "Reserved").
-- **Assign a Rack or RackGroup**: Specify the rack or rack group associated with the tile.
-- **Specify Rack Orientation**: Define the orientation of the rack relative to the floor plan.
+- **Assign a Object or RackGroup**: Specify the object or rack group associated with the tile.
+- **Specify Rack Orientation**: Define the orientation of the object relative to the floor plan.
 - **Adjust Tile Size**: Expand a tile to cover multiple spaces.
   - Useful for documenting larger-than-usual racks or marking sections of the floor plan as "Reserved" or "Unavailable."
 
 !!! note
-    Once a Rack has been placed on a **Floor Plan Tile** you cannot update the **Location** of the Rack until it has been removed from the Floor Plan or the Floor Plan has been deleted.
+    Once a Object has been placed on a **Floor Plan Tile** you cannot update the **Location** of the Object until it has been removed from the Floor Plan or the Floor Plan has been deleted.
+
+![Add Tile form part 1](../images/add-tile-form.png)
 
 ### Working with RackGroup and Status Tiles
 
 When tiles cover multiple spaces:
 
-- You can place racks within these tiles.
-- For RackGroup tiles, racks must be added to the appropriate RackGroup.
+- You can place objects within these tiles.
+- For RackGroup tiles, objects must be added to the appropriate RackGroup before they can be placed on a tile.
```

Comment:
Same as above, objects is vague here.

Reply:
I see just below this is defined, Maybe move line 182 above these two and call out that they are the objects that can be added to the Floor Plan.